### PR TITLE
Bug 1571409 - Add a "My alert summaries" checkbox on Alerts view

### PR DESCRIPTION
**Pull request by octavian-negru**

### CODE REVIEW

File: ui/perfherder/alerts/AlertTable.jsx [hunk 1]
```diff
@@ -85,20 +85,38 @@ export default class AlertTable extends React.Component {
...
+    const notDownstreamFromSummary =
+      alert.summary_id === alertSummary.id ||
+      alert.status !== alertStatusMap.downstream;
+    const isReassigned =
+      hideDownstream &&
+      alert.status === alertStatusMap.reassigned &&
+      alert.related_summary_id !== alertSummary.id;
+    const isDownstream =
+      hideDownstream && alert.status === alertStatusMap.downstream;
+    const isValid = hideDownstream && alert.status === alertStatusMap.invalid;
```
Comment (airimovici) [inline ui/perfherder/alerts/AlertTable.jsx]: name it isNotValid

**Comment by ionutgoldan** [comment]
I'm commandeering Octavian's PR & until he gets back.

**Comment by ionutgoldan** [comment]
Some minor changes were needed to make this feature work as expected.

**Comment by ionutgoldan** [comment]
@sarah-clements I've kept 2 commits on this PR to distinguish Octavian's work from mine.
You can treat them as a single unit.

**Comment by sarah-clements** [comment]
Cam is going to review this one :)

**Comment by ionutgoldan** [comment]
FYI @karlht

**Comment by camd** [comment]
I will review this PR today.

**Comment by ionutgoldan** [comment]
> This looks good to me. I tested it locally a bit and it seems to work well. It's not totally obvious that it's necessary to hit `enter` after clicking "TAKE". an OK, button might be good to add at some point. But not necessary here.

Yes, I agree. We talked about this in the FxPerfTest demo preview we had last week. It could have been done better, but we decided to land this and rather wait for the Perf sheriffs' feedback on how to precisely tweak this.

**Comment by ionutgoldan** [comment]
Thanks, Cameron!

**Pull request merged**